.github/CODEOWNERS

```diff
@@ -9,14 +9,12 @@
 
 # File type specific owners
 # Markdown specific files
-*.md    @awentzel @chrisdholt @falkicon
+*.md @awentzel @chrisdholt @falkicon
```

Comment:
It looks like this indenting was originally intended to match with the indenting on line 8, can you also adjust the indenting on line 8?

Reply:
Your suggestion for consistency was good, however, I instead used one space after each path, which was more consistent with the majority of the rules in the file.